zfit/z/numpy.py

```diff
@@ -10,14 +10,140 @@
 
 #  Copyright (c) 2024 zfit
 
-import tensorflow as tf
+import numpy as _np
+import tensorflow as _tf
 from tensorflow.experimental.numpy import *
 
 
 class linalg:
-    inv = staticmethod(tf.linalg.inv)
-    det = staticmethod(tf.linalg.det)
-    solve = staticmethod(tf.linalg.solve)
+    inv = staticmethod(_tf.linalg.inv)
+    det = staticmethod(_tf.linalg.det)
+    solve = staticmethod(_tf.linalg.solve)
 
 
-del tf
+def faddeeva_humlicek(z, s=10.0):
```

Comment:
few things: it's probably good to move it under a "special" namespace, but we don't have that right now. For future, maybe make a comment?
document `s` (and maybe also replace it in the formula below?)

Reply:
Fixed. Is there a good place to also add the Faddeeva function in the docs?

Reply:
Hm, there is currently no default case, but that's a bit historic as the "new backend", znp, is introduced as a "works if works", as it's still experimental in TF, technically. But it was so stable, factually, that we exposed it.
That would need to add a backend API docs file, if you want to try, please go ahead and add what you think is sensibse (that would mean to add the zfit.z.numpy in the `user_api`, otherwise, I'll note it down for the docs overhaul